Artem-Ushenko/Portfolio_Web_Static_On_AWS_S3_Bucket
# Add Terraform Configuration for Automated S3 Bucket and Static Website Setup

### Summary
This pull request introduces new Terraform configurations that automate the creation of an S3 bucket designed to host static website content. The setup includes configuring the bucket to serve web pages and ensuring it has the correct policies in place to allow public access.

### Changes
- Added `terraform`: Contains the Terraform code to create an S3 bucket with the necessary permissions and properties to host a static website.

### How to Test
1. Ensure you have Terraform installed and AWS CLI configured with appropriate credentials.
2. Navigate to the directory containing the new Terraform files.
3. Initialize the Terraform environment:
   ```bash
   terraform init
4. Apply the Terraform configuration:
   ```bash
   terraform apply
5. Confirm that the S3 bucket is created, and the index.html is accessible via the bucket's public URL.

### Additional Notes
Ensure that your AWS account has the necessary permissions to create S3 buckets and apply bucket policies.
Review the bucket policy applied by this configuration to ensure it matches your security requirements.
